.gitignore

```diff
@@ -6,4 +6,19 @@ dockerfiles/run/**/keystore
 dockerfiles/run/**/glowroot
 .m2
 test-run.log
-build
\ No newline at end of file
+build
+*.iml
+.idea
+*.lib
+*.jar
+.floo
+.flooignore
+.m2
+_site/
+keystore
+derby.log
+src/homepage/Gemfile.lock
+mailbox/var/
+src/homepage/.sass-cache/
+KahaDB/
+.attach_pid*
```

Comment:
Please use git rebase to move changes of `.gitignore` in a distinct commit.

Reply:
I asked him to set this as global git ignore instead, so I think this change can be removed.

Reply:
Still not removed

Reply:
Still not removed